src/api/app/assets/stylesheets/webui/build-results.scss

```diff
@@ -22,7 +22,6 @@
   a[data-toggle="collapse"] { width: 1.875rem; }
 }
 
-// TODO: move to layout.scss as soon as the responsive_ux layout is merged
```

Comment:
This comment isn't true since it's only affecting the build results and live build logs. It doesn't belong in the layout.